Packs/CommonScripts/Scripts/ReadQRCode/ReadQRCode.py

```diff
@@ -1,35 +1,38 @@
 import demistomock as demisto  # noqa
 from CommonServerPython import *  # noqa
+from pyzbar import pyzbar
 import cv2
 # pylint: disable=E1101  # disable pylint not recognizing cv2's attributes.
 
 
-def read_qr_code(filename: str) -> str:
+def read_qr_code(filename: str) -> str | list[str]:
```

Comment:
dont return a string or a list. 
a single string should be put into a list of 1

Reply:
what's the point? it's extra work and the output is exactly identical

Reply:
its bad practice to have a variable return type to your functions